newbank/server/NewBank.java

```diff
@@ -127,11 +116,13 @@ private String createLoginAccount(String request) {
 			String output = "The username already exists.\nPlease enter a unique username or type 'esc' to return to the menu screen.";
 			return output;
 		} else {
-			Customer newCustomer = new Customer(customerName, userName, password, iban);       // create new customer
+			Customer newCustomer = new Customer(customerName, userName, password);       // create new customer
 			newCustomer.addAccount(new Account("Main", 00.0));    // create a default account for the customer
 			bank.customers.put(userName, newCustomer);        // add the customer to the list of customers and assign their username
+			Account newAccount = newCustomer.getAccount("Main");
+			String newIBAN = newAccount.getIBAN();
 			String output = "New user '" + userName + "' created.\n" +
-					"We also created an initial bank account:'Main' with the IBAN: " + iban + " for you.\n" +
+					"We also created an initial bank account:'Main' with the IBAN: " + newIBAN + " for you.\n" +
 					"Please Download the Google Authenticator App and use the key NY4A5CPJZ46LXZCP to set up your 2FA.";
```

Comment:
```suggestion
					"An initial bank account:'Main' with the IBAN: " + newIBAN + " has been also created for you.\n";
```

Can you delete this bit, it's a skeleton output of something that I didn't have time to implement in the previous sprint.